src/main.js

```diff
@@ -34,24 +33,15 @@ let load = loader({
   },
 
   monitor: {
-    requires: ['cfg', 'drain'],
-    setup: ({cfg, drain}) => base.stats.startProcessUsageReporting({
-      drain:      drain,
-      component:  cfg.app.statsComponent,
-      process:    'server'
+    requires: ['cfg', 'drain', 'process'],
+    setup: ({cfg, process}) => base.monitor({
+      project:      'taskcluster-auth',
+      credentials:  cfg.taskcluster.credentials,
```

Comment:
This is still going to be our biggest issue because these don't exist. We need to figure out a way to get tc-lib-validate to work without these.